packages/features/ee/organizations/pages/components/OtherTeamListItem.tsx

```diff
@@ -38,7 +37,7 @@ export default function OtherTeamListItem(props: Props) {
 
   const orgBranding = useOrgBranding();
 
-  const isOwner = props.team.role === MembershipRole.OWNER;
+  const isOwner = true; // Based on trpc auth procedures this endpoint is only accessible to org owners|admins
```

Comment:
Probably I should remove this flag and then it's implementation as it no longer be conditional.

Reply:
I agree, it would simplify the code as well

Reply:
Some observations:
- I think we should prefix `listOtherTeamMembers` with admin to make it very obvious. 
- `OtherTeamListItem` isn't aware how the data is fetched, so, we shouldn't take decision based on that here. Might be better to still keep `isOwner` flag so that the component can be reused for both Admin and Non admin members if needed.

Reply:
Agree on rename listOtherTeamMembers, good call. Will address it on follow up PR.
About keeping isOwner flag: being owner of an other team listing will never happen as it then will be listed on your normal team list and then the component won't be required because if admin|owner of that team it will not be on other team list.